Fix memory profiling for memory.view ops

kimishpatel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #7927
* #7926
* __->__ #7925

ATT

Differential Revision: [D68448333](https://our.internmc.facebook.com/intern/diff/D68448333/)

cc @Olivia-liu @Jack-Khuu

kimishpatel committed:
Fix memory profiling for memory.view ops

ATT

Differential Revision: [D68448333](https://our.internmc.facebook.com/intern/diff/D68448333/)

[ghstack-poisoned]

Jack-Khuu reviewed (approved):
Accepted internally, mirroring stamp

kimishpatel committed:
Update on "Fix memory profiling for memory.view ops"


ATT

Differential Revision: [D68448333](https://our.internmc.facebook.com/intern/diff/D68448333/)

cc Olivia-liu Jack-Khuu

[ghstack-poisoned]

kimishpatel committed:
Update on "Fix memory profiling for memory.view ops"


ATT

Differential Revision: [D68448333](https://our.internmc.facebook.com/intern/diff/D68448333/)

cc Olivia-liu Jack-Khuu

[ghstack-poisoned]
